amplify/backend.ts

```diff
@@ -6,8 +6,12 @@ import { storage } from './storage/resource';
 /**
  * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions, and more
  */
-defineBackend({
+const backend = defineBackend({
   auth,
   data,
   storage
 });
+
+backend.data.resources.cfnResources.cfnApiKey?.overrideLogicalId(
+  `recoverApiKey${new Date().getTime()}`
```

Comment:
Just checking this new Date().getTime() will regenerate the API key on every deploy ?